lib/engine/g_18_zoo/corporation.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
```

Comment:
This subclass does not seem to have a point

Reply:
yes, you're right, removing it